packages/graphics/src/GraphicsGeometry.ts

```diff
@@ -404,7 +444,7 @@ export class GraphicsGeometry extends BatchGeometry
      * Generates intermediate batch data. Either gets converted to drawCalls
      * or used to convert to batch objects directly by the Graphics object.
      */
-    updateBatches()
+    updateBatches(): void
```

Comment:
Should this have public in front of it or is it meant to be protected but cant because its used in `Graphics`?

Reply:
Internal